config-path.js

```diff
@@ -0,0 +1,29 @@
+const path = require('path');
+const userHome = require('user-home');
+
+const env = process.env;
+const name = 'js-v8flags';
+
+function macos () {
+	const library = path.join(userHome, 'Library');
+	return path.join(library, 'Caches', name);
+}
+
+function windows () {
+	const appData = env.LOCALAPPDATA || path.join(userHome, 'AppData', 'Local');
+	return path.join(appData, name, 'Cache');
+}
+
+// https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html
+function linux () {
+	const username = path.basename(userHome);
+	return path.join(env.XDG_CACHE_HOME || path.join(userHome, '.cache'), name);
+}
+
+if (process.platform === 'darwin') {
```

Comment:
We aren't dropping 0.10 support because it is still the latest node version shipped in official channels on some linux.

I don't like returning the function; we should really just figure out what is going on here.